Doc/library/pathlib.rst

```diff
@@ -552,55 +552,54 @@ Pure paths provide the following methods and properties:
       PureWindowsPath('c:/Program Files')
 
 
-.. method:: PurePath.match(pattern, *, case_sensitive=None)
+.. method:: PurePath.globmatch(pattern, *, case_sensitive=None)
```

Comment:
Use an underscore for readability and accessibility?

```suggestion
.. method:: PurePath.glob_match(pattern, *, case_sensitive=None)
```

Reply:
A couple other options: 

- `match_glob()` has a slightly more natural word order
- `full_match()` aligns with `re.fullmatch()`.

Reply:
Yes, they sound good too.